static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
Always attempt to put methods as far down the hierarchy as possible, so give ContactsCtrl a try. The catch is if you set a variable it'll change in the copy attached to the $scope of ContactsCtrl, not InboxCtrl. For example, `$scope.items = []` wouldn't update the parent scope. This is why we have setters in InboxCtrl - so the child controllers can update the parent scope. Splice should be fine, but it's something to keep an eye on.